elliottlib/cli/find_builds_cli.py

```diff
@@ -142,30 +161,18 @@ def _determine_errata_info(runtime):
     if not runtime.branch:
         raise ElliottFatalError('Need to specify a branch either in group.yml or with --branch option')
     base_tag = runtime.branch
-
     et_data = runtime.gitdata.load_data(key='erratatool').data
-    product_version = override_product_version(et_data.get('product_version'), base_tag)
     tag_pv_map = et_data.get('brew_tag_product_version_mapping')
-    return base_tag, product_version, tag_pv_map
+    return base_tag, tag_pv_map
```

Comment:
I don't think there is any need to use `runtime.branch` after we have the tag_product_version_mapping,

Reply:
yeah, removed.